docs/iris/src/whatsnew/contributions_1.8/newfeature_2014-12-19_netcdf-streaming.txt

```diff
@@ -1,4 +1,5 @@
 * NetCDF saving to all NetCDF4/HDF encodings (including the default) now uses biggus save when the cube has lazy data.
+
```

Comment:
Both of the whatsnew examples included here are _far_ too long. Whatsnew entries should be concise! You should be able to summarise the whole change in a maximum of two sentences, not three bullet points.

This newfeature is an exception to the above as it actually ended up going into the 1.8 whatsnew as two bullet points. In light of this, please can the text here be replaced with what is in the 1.8 whatsnew.